marketing_security_group/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==============================
+Marketing security extra rules
+==============================
+
+This module add a this security features:
+
+* ACL for allowing marketing user group to remove mass.mailing objects
+* Security rule to allows marketing user to delete only his not sent mail.mass_mailing objects
+* Security rule to allows marketing manager to delete all not sent mail.mass_mailing objects
+
+For sent mail.mass_mailing objects, only an user in Administration / Settings group
```

Comment:
This is correct, isn't it? In english we use ```an``` when next noun starts with a vocal.